content/events/2022-11-24/quest_to_the_fake_object.html

```diff
@@ -0,0 +1,475 @@
+<!doctype html>
```

Comment:
@sbauza could you rename your HTML file to `slides.html`? All the other events do have the same layout

Reply:
Done